packages/util/package.json

```diff
@@ -90,10 +90,11 @@
     "tsc": "../../config/cli/ts-compile.sh"
   },
   "dependencies": {
-    "@ethereumjs/rlp": "^5.0.1",
-    "ethereum-cryptography": "^2.1.2"
+    "@ethereumjs/rlp": "^5.0.0",
+    "abstract-level": "^1.0.3",
```

Comment:
Can we make both of the level deps optional peer dependencies like kzg instead of required?  Otherwise, we're shipping these deps to essentially everyone who uses any component of our stack?

Reply:
Fixed.